### PR TITLE
add guard to return early

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- return early if no locations are passed along to query by when getting quantities for inventory items

**Why**
- `where location_id in ()` is not valid sql syntax and causes a 500 to be raised.
- if there are no specified locations the result should be 0

Fixes CORE-1178

### CODE REVIEW

File: packages/inventory/src/services/inventory-level.ts [hunk 1]
```diff
@@ -221,6 +221,10 @@ export default class InventoryLevelService extends TransactionBaseService {
       locationIds = [locationIds]
     }
 
+    if(locationIds.length === 0) {
```
Comment (adrien2p) [inline packages/inventory/src/services/inventory-level.ts:224]: suggestion: `if (!locationIds.length)`

File: packages/inventory/src/services/inventory-level.ts [hunk 2]
```diff
@@ -248,6 +252,10 @@ export default class InventoryLevelService extends TransactionBaseService {
...
```
Comment (adrien2p) [inline packages/inventory/src/services/inventory-level.ts:255]: suggestion: `if (!locationIds.length)`

File: packages/inventory/src/services/inventory-level.ts [hunk 3]
```diff
@@ -275,6 +283,10 @@ export default class InventoryLevelService extends TransactionBaseService {
...
```
Comment (adrien2p) [inline packages/inventory/src/services/inventory-level.ts:286]: suggestion: `if (!locationIds.length)`

**Review by adrien2p — CHANGES_REQUESTED** [review]

**Comment by srindom** [comment]
@pKorsholm can you take a look at @adrien2p's comments so we can get this merged :)

**Review by adrien2p — APPROVED** [review]

**Pull request merged**